<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>821</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Struct Init With Init List:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textInit">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Struct Init With Fields:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textFileds">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Engine Cylinders: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textCylinders"/>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Engine Volume: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textVolume"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Tire Mark: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxTieMark">
           <item>
            <property name="text">
             <string>S</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>H</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QRadioButton" name="radioButtonTransmission">
         <property name="text">
          <string>Manual Transmission</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Car Number: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textNumber"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Years Of Maintenance: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textYears"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Input your Structure:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPrintStrcut">
         <property name="text">
          <string>Print</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Your Struct: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textUser">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Pointer Struct:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textPointer">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Reference Struct: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textRef">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
